Tests/Routing/Loader/RestYamlCollectionLoaderTest.php

```diff
@@ -184,6 +214,36 @@ public function testControllerAsServiceWithClassName()
     }
 
     /**
+     * Test that YAML collection with named prefixes gets parsed correctly with inheritance.
+     */
+    public function testNamedPrefixedBaseReportsFixture()
+    {
+        $collection = $this->loadFromYamlCollectionFixture('base_named_prefixed_reports_collection.yml');
+        $etalonRoutes = $this->loadEtalonRoutesInfo('base_named_prefixed_reports_collection.yml');
+
+        foreach ($etalonRoutes as $name => $params) {
+            $route = $collection->get($name);
+            $methods = $route->getMethods();
+
+            $this->assertNotNull($route, $name);
+            $this->assertEquals($params['path'], $route->getPath(), $name);
+            $this->assertEquals($params['method'], $methods[0], $name);
+            $this->assertContains($params['controller'], $route->getDefault('_controller'), $name);
+        }
+
+        $name = 'api_get_billing_payments';
+        $this->assertArrayNotHasKey($name, $etalonRoutes);
+    }
+
+    public function testRoutesWithPattern()
```

Comment:
It looks like this one should be deprecated (as it was deprecated in symfony) but ok for this test.